sdks/java/io/hadoop-input-format/jdk1.8-hifio-tests/src/test/resources/kubernetes/elasticsearch/es-discovery-service.yaml

```diff
@@ -0,0 +1,31 @@
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+# Deployment script for discovering Elasticsearch nodes.
```

Comment:
This comment is misleading - it implies that the script itself finds new elastic search, which isn't what this script does.